### PR TITLE
Add ItemAlteration rule element to Explosive Arrival

### PR DESCRIPTION
**Pull request by ammalagonc**
This would proc a note on Summon spells if a Wizard PC has the Explosive Arrival Feat.

Note - damage does not scale from the note.

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -4367,7 +4367,8 @@
                 },
                 "UnifiedMagicalTheory": {
                     "Prompt": "Select a 1st-level wizard feat."
-                }
+                },
+                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[1d4[fire]] damage per spell rank. If the creature summoned has the acid, cold, electricity, sonic, or spirit trait, you can deal damage of that type instead."
```
Comment (pedrogrullada) [inline static/lang/re-en.json:4371]:
```suggestion
                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[@item.level[fire]] damage. If the creature summoned has the acid, cold, electricity, sonic, or spirit trait, you can deal damage of that type instead."
```

**Comment by ammalagonc** [comment]
Do you think there's a way to account for the change in trait?

**Comment by pedrogrullada** [comment]
> Do you think there's a way to account for the change in trait?

There's a few ways to go about it. The easiest might be to make those damage type mentions into damage links.

```
If the creature summoned has the @Damage[@item.level[acid]]{acid},
```

File: static/lang/re-en.json [hunk 2]
```diff
@@ -4367,7 +4367,8 @@
                 },
                 "UnifiedMagicalTheory": {
                     "Prompt": "Select a 1st-level wizard feat."
-                }
+                },
+                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[@item.level[fire]] damage. If the creature summoned has the acid, cold, electricity, sonic, or spirit trait, you can deal damage of that type instead."
```
Comment (TikaelSol) [inline static/lang/re-en.json:4371]:
```suggestion
                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[(@item.level)d4[fire]] damage. If the creature summoned has the @Damage[(@item.level)d4[acid]]{acid}, @Damage[(@item.level)d4[cold]]{cold}, @Damage[(@item.level)d4[electricity]]{electricity}, @Damage[(@item.level)d4[sonic]]{sonic}, or @Damage[(@item.level)d4[spirit]]{spirit} trait, you can deal damage of that type instead."
```

**Comment by TikaelSol** [comment]
This should be a toggle with a mergeable suboption. Even if you will use this 99% of the time with summon spells you may choose not to or maybe some other free action spellshape action is tempting.

File: static/lang/re-en.json (hunk 2 above)
Comment (TikaelSol) [inline static/lang/re-en.json:4371]: Let's also move this under a spellshape section, so it is `PF2E.SpecificRule.Spellshape.ExplosiveArrival`

**Comment by ammalagonc** [comment]
Incorporated all of the suggested changes. Please review when you get a moment. 

Thanks!

File: packs/feats/explosive-arrival.json [hunk 3]
```diff
@@ -24,7 +24,38 @@
...
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Spellshape.ExplosiveArrival"
+                    }
+                ]
+            },
+            {
+                "key": "RollOption",
+                "label": "PF2E.SpecificRule.Spellshape.Label",
```
Comment (pedrogrullada) [inline packs/feats/explosive-arrival.json:45]:
```suggestion
                "label": "PF2E.TraitSpellshape",
```

File: static/lang/re-en.json [hunk 4]
```diff
@@ -3871,6 +3871,10 @@
                 "Tenth": "10th-Rank",
                 "Third": "3rd-Rank"
             },
+            "Spellshape": {
+                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[(@item.level)d4[fire]] damage. If the creature summoned has the @Damage[(@item.level)d4[acid]]{acid}, @Damage[(@item.level)d4[cold]]{cold}, @Damage[(@item.level)d4[electricity]]{electricity}, @Damage[(@item.level)d4[sonic]]{sonic}, or @Damage[(@item.level)d4[spirit]]{spirit} trait, you can deal damage of that type instead.",
+                "Label": "Spellshape"
```
Comment (pedrogrullada) [inline static/lang/re-en.json:3876]: No need for this one, we already have a localized label for the Spellshape trait in `en.json` :)

File: packs/feats/explosive-arrival.json (hunk 3 above)
Comment (TikaelSol) [inline packs/feats/explosive-arrival.json:45]:
```suggestion
                "label": "PF2E.TraitSpellshape",
```

File: static/lang/re-en.json (hunk 4 above)
Comment (TikaelSol) [inline static/lang/re-en.json:3876]:
```suggestion
                "ExplosiveArrival": "Your summoned creature appears in a detonation of arcane runes. If your next action is to Cast a Spell with the summon trait, all creatures in a @Template[type:emanation|distance:10] around the creature you summoned take @Damage[(@item.level)d4[fire]] damage. If the creature summoned has the @Damage[(@item.level)d4[acid]]{acid}, @Damage[(@item.level)d4[cold]]{cold}, @Damage[(@item.level)d4[electricity]]{electricity}, @Damage[(@item.level)d4[sonic]]{sonic}, or @Damage[(@item.level)d4[spirit]]{spirit} trait, you can deal damage of that type instead."
```

**Pull request merged**